### ISSUE TITLE
A11y_Visual Studio Code Services_Docs_Left Right Navigation_Resize: After zooming the page up to 200%, right navigation 'Getting started' links are not visible.

### ISSUE THREAD

**Issue by SachinChavhan317** [microsoft/vscode-docs#6205]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**
### GitHubTags:
#WCAG1.4.4; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #Resize Text;

### Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 111.0.5563.147 (Official Build) (64-bit)

### Repro Steps:
1. Hit the application URL: https://code.visualstudio.com/
2. Navigate to "Docs" link.
3. Left and right navigation will appear.
4. Verify whether the after zooming the page up to 200%, right navigation 'Getting started' links are visible or not.

### Actual Result:
After zooming the page up to 200%, right navigation 'Getting started' links are not visible.

### Expected Result:
After zooming the page up to 200%, right navigation 'Getting started' links should be visible on page.

### User Impact:
Low Vision users will not get this right navigation feature links if getting started links are not visible in 200% zoom.

### Attachment:
![13](https://user-images.githubusercontent.com/86241494/230283244-a436347a-9ada-4f08-9886-a1a7849918ed.gif)

**Comment by rzhao271** [comment]
Hi @SachinChavhan317, I have attached a screenshot of a new design. It has not been published yet. Let me know if the design works for you:
![Two selectboxes near the top of the Blogs page when the page width is too narrow to display the right nav](https://github.com/microsoft/vscode-docs/assets/7199958/ab4c03eb-db05-4ee1-9508-d0097ac85b68)

The design adds a new selectbox to the top of the page when the width of the page is small. The new selectbox contains the names of the on-page navigation headers, and is right under another selectbox that controls the page the reader is on.

CC @gregvanl

**Comment by SachinChavhan317** [comment]
I have verified the bug on URL: https://code.visualstudio.com/docs
I'm not getting any selectbox at the top of the screen for right navigation.
Please refer below attachment for reference.
![6205_Resize](https://github.com/microsoft/vscode-docs/assets/86241494/04bdedf0-31b7-443f-b9b7-119417f0e9c0)

**Comment by rzhao271** [comment]
@SachinChavhan317 I forgot to apply the fix to a few pages. The issue should be fixed for that page too, now.

**Comment by rzhao271** [comment]
Closing due to upcoming SLA deadline.

**Comment by SachinChavhan317** [comment]
Verified the bug on URL: https://code.visualstudio.com/docs
Getting started dropdown is available for 'Getting started' controls. But below controls still not available in resize resolution.

Adding this screenshot for reference.
![image](https://github.com/microsoft/vscode-docs/assets/86241494/0a278ef5-d230-49fb-a449-ff35851fcf9d)

**Comment by rzhao271** [comment]
Hi @SachinChavhan317, I have added some updates to the site that have to do with your screenshot in https://github.com/microsoft/vscode-docs/issues/6210#issuecomment-1576519542. Please take a look again and let me know whether this bug can be closed.

We have removed the "Tweet this Link" link from all pages.
After reflow, the links are at the bottom of the page. The selectboxes at the top are only for navigating within a section of the website or to another part of the page.

**Comment by SachinChavhan317** [comment]
Verified the bug on URL: https://code.visualstudio.com/docs
The issue is fixed, hence closing the bug.
Adding fixed attachment for reference.
#Closed;
![Presentation2](https://github.com/microsoft/vscode-docs/assets/86241494/b4b06493-d87f-4a31-a452-5002baa8ea82)
